lib/jxl/enc_fast_lossless.cc

```diff
@@ -715,6 +725,34 @@ void JxlFastLosslessPrepareHeader(JxlFastLosslessFrameState* frame,
   output->ZeroPadToByte();  // Groups are byte-aligned.
 }
 
+#if !FJXL_STANDALONE
+void JxlFastLosslessOutputAlignedSection(
```

Comment:
Doesn't WriteOutput already have a special case for this case? or am I missing something?

Reply:
The reason I did this was that WriteOutput always requested at least 32 bytes, and then the actual bytes written was less. This works if we write the output sequentially, but when we seek back and write the header, sometimes requesting at least 32 bytes is interpreted as trying to overwrite an already written output and we get an assert failure.

Reply:
Ah, that is indeed annoying... I wonder if there is a better solution than this, for example relaxing the WriteOutput requirements, but if nothing else comes to mind then that's fine with me.

Reply:
Not without making Write output more complex, and it is complex enough as it is. I would be in favor of handling the simple case with this straightforward function.

Reply:
Fair enough. Maybe we can just add a TODO for the future...